jsonnet/rules.libsonnet

```diff
@@ -301,11 +301,11 @@ local droppedKsmLabels = 'endpoint, instance, job, pod, service';
             // Returns 1 if all control plane nodes are ready and is absent otherwise. Should be used to suppress alerts during control plane upgrades or disruption.
           },
           {
-            expr: 'rate(cluster_monitoring_operator_reconcile_errors_total[15m]) * 100 / rate(cluster_monitoring_operator_reconcile_attempts_total[15m]) > 10',
+            expr: 'max_over_time(cluster_monitoring_operator_last_reconciliation_successful[30m]) == 0',
```

Comment:
nit but as it stands the alert would fire after 30m (range duration) + 30m (for duration). It might be more readable to use a range duration of 1h and remove the "for" duration?

Reply:
indeed, good suggestion, changed the rule accordingly